pandas/io/parsers.py

```diff
@@ -612,10 +612,9 @@ def read_csv(
     # Thus, we need a flag to indicate that we need to "override"
     # the comparison to dialect values by checking if default values
     # for BOTH "delimiter" and "sep" were provided.
-    default_sep = ","
```

Comment:
You have to set sep="," in default case when ``delim_whitespace`` is False

Reply:
I am setting it on line 632. I don't want to propagate the `lib.no_default`  object into `_read`, so I changed it to the default value disregarding `delim_whitespace` value. Thus, it will be `_read(delimiter=",", ...)` in the say way as it was done before. If you think that it is safe to propagate `lib.no_default` further, I can change it and check it

Reply:
Sorry missed that